make apply-k8s-utils fails

```
aws eks --region us-west-2 update-kubeconfig --name test99-production-us-west-2
Updated context arn:aws:eks:us-west-2:514226198018:cluster/test99-production-us-west-2 in /Users/trevoro/.kube/config
pushd kubernetes/terraform/environments/production; \
	terraform init; \
	terraform apply
~/code/ops/kubernetes/terraform/environments/production ~/code/ops
[0m[1mInitializing modules...[0m

[0m[1mInitializing the backend...[0m

[0m[1mInitializing provider plugins...[0m

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.helm: version = "~> 1.1"
* provider.kubernetes: version = "~> 1.11"
* provider.local: version = "~> 1.4"
* provider.null: version = "~> 2.1"
* provider.template: version = "~> 2.1"

[0m[1m[32mTerraform has been successfully initialized![0m[32m[0m
[0m[32m
You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.[0m


Error: Missing required argument
The argument "region" is required, but was not set.
make: *** [apply-k8s-utils] Error 1
```

the contents of the `main.tf` file in the environment directory:

```
terraform {
  backend "s3" {
    bucket         = "test99-production-terraform-state"
    key            = "infrastructure/terraform/environments/production/kubernetes"
    encrypt        = true
    region         = "us-west-2"
    dynamodb_table = "test99-production-terraform-state-locks"
  }
}

# Provision kubernetes resources required to run services/applications
module "kubernetes" {
  source = "../../modules/kubernetes"

  environment = "production"
  region      = "us-west-2"

  # Authenticate with the EKS cluster via the cluster id
  cluster_name = "test99-production-us-west-2"

  external_dns_zone = "czerotest.com"
  external_dns_owner_id = "efc2dd49-c971-4ef1-ae5b-8b786d3fc59f" # randomly generated ID

  # Registration email for LetsEncrypt
  cert_manager_acme_registration_email = "devops@czerotest.com"
}
```

--- a/kubernetes/terraform/environments/production/main.tf
+++ b/kubernetes/terraform/environments/production/main.tf
@@ -8,6 +8,10 @@ terraform {
   }
 }
 
+provider "aws" {
+  region  = "<% index .Params `region` %>"
+}
+
 # Provision kubernetes resources required to run services/applications
 module "kubernetes" {
   source = "../../modules/kubernetes"

